### PR TITLE
kvm/security_group: Make Security Group Python 3 compatible

### PR DESCRIPTION
**Pull request by wido**
This script only runs on the KVM Hypervisors and these all support
Python 3.

As Python 2 is deprecated at the end of 2019 we need to fix these
scripts to work under Python 3.

CentOS 7, 8 and Ubuntu 16.04 and 18.04 all have Python 3 installed
by default.

Ubuntu 20.04 will no longer have Python 2 installed and therefor
this script needs to be modified to work with Python 3.

Signed-off-by: Wido den Hollander <wido@widodh.nl>

### CODE REVIEW

**Comment by GabrielBrascher** [comment]
Just adding, for the record, that this is the first PR addressing the effort raised on issue #3195 
Thanks for the PR @wido! I will be reviewing it and re-testing it.

**Comment by rhtyd** [comment]
@wido @GabrielBrascher can you add dependency of `python3` in `packaging/centos7/cloud.spec` to cloudstack-common and cloudstack-agent packages?

File: debian/control [hunk 1]
```diff
@@ -3,13 +3,13 @@ Section: libs
...
 Maintainer: Wido den Hollander <wido@widodh.nl>
 Build-Depends: debhelper (>= 9), openjdk-8-jdk | java8-sdk | java8-jdk | openjdk-9-jdk, genisoimage,
- python-mysql.connector, maven (>= 3) | maven3, python (>= 2.7), lsb-release, dh-systemd, python-setuptools
+ python-mysql.connector, maven (>= 3) | maven3, python (>= 2.7), python3 (>= 3), lsb-release, dh-systemd, python-setuptools
 Standards-Version: 3.8.1
 Homepage: http://www.cloudstack.org/
 
 Package: cloudstack-common
 Architecture: all
-Depends: ${misc:Depends}, ${python:Depends}, genisoimage, nfs-common, python-netaddr
```
Comment (rhtyd) [inline debian/control]: @wido  @GabrielBrascher Do we no longer need python-netaddr? (for all/any py scripts?)

File: packaging/centos7/cloud.spec [hunk 2]
```diff
@@ -54,7 +54,7 @@ intelligent IaaS cloud implementation.
 %package management
 Summary:   CloudStack management server UI
 Requires: java-1.8.0-openjdk
-Requires: python
+Requires: python3
```
Comment (rhtyd) [inline packaging/centos7/cloud.spec]: We probably want both python and python3 like the debian/control change @GabrielBrascher, please add a new Requires line

**Comment by rhtyd** [comment]
Thanks @GabrielBrascher left some minor remarks

File: packaging/centos7/cloud.spec (hunk 2 above)
Comment (GabrielBrascher) [reply packaging/centos7/cloud.spec]: @rhtyd ah sure. Updated it, thanks!

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan test

**Pull request merged**